planning/motion_velocity_planner/autoware_motion_velocity_planner_node/src/planner_manager.cpp

```diff
@@ -71,13 +71,73 @@ void MotionVelocityPlannerManager::update_module_parameters(
   for (auto & plugin : loaded_plugins_) plugin->update_parameters(parameters);
 }
 
+std::shared_ptr<DiagnosticStatus> MotionVelocityPlannerManager::makeDiagnostic(
+  const std::string & reason,
+  const bool is_decided)
+{
+  auto status = std::make_shared<DiagnosticStatus>();
+  status->level = status->OK;
+  status->name = reason;
+  diagnostic_msgs::msg::KeyValue key_value;
+  {
+    // Decision
+    key_value.key = "decision";
+    if (is_decided)
+      key_value.value = reason;
```

Comment:
desicion must be  "none" or "deceleration" 

![image](https://github.com/user-attachments/assets/40af18d5-707d-4e14-ac3c-044c50a3a47d)

Reply:
sorry, I misunderstood,
obstastacle cruise planner has other value too. 
https://github.com/tier4/autoware.universe/blob/feat/steer_wheel_sim/planning/autoware_obstacle_cruise_planner/src/planner_interface.cpp#L874-L891

And planning_evaluator will change to `deceleration` if not `none`

Reply:
obstastacle cruise planner publish stop position as value.

![image](https://github.com/user-attachments/assets/fdd5ec8b-d924-432c-b8ca-302530c412ad)

Reply:
In current Implementation, the name and value is same. Do you have any idea about better infomation for value.
If no I think it is ok just `stop` or `deceleration` ..